### ISSUE TITLE
"Clear choices on this page" option or similar

### ISSUE THREAD

**Issue by emilymbender** [delph-in/matrix#629]
In the case where people are working with e.g. automatically inferred choices files, it would be nice to be able to clear portions of the questionnaire (e.g. arg-opt) without having to resort to hand-editing the choices file. In other words, where we have radio buttons, there should be the possibility of saying "none of these".

**Comment by olzama** [comment]
Yes, there is already "Clear current subpage", but I suppose you mean, portions of the subpage? I remember often needing that.

**Comment by emilymbender** [comment]
Ah, good point. Yes, I think the students in this case would like to be able to clear specific selections, without losing everything on the page.
